templates/web/base/report/form/user_loggedout.html

```diff
@@ -0,0 +1,53 @@
+[% IF c.cobrand.social_auth_enabled %]
+  <div class="js-new-report-user-hidden">
+  [% IF c.config.FACEBOOK_APP_ID %]
+    <div class="form-box">
+    <button name="facebook_sign_in" id="facebook_sign_in" value="facebook_sign_in" class="btn btn--block btn--social btn--facebook">
+        <img alt="" src="/i/facebook-icon-32.png" width="17" height="32">
+        [% loc('Log in with Facebook') %]
+    </button>
+    </div>
+  [% END %]
+  [% IF c.config.TWITTER_KEY %]
+    <div class="form-box">
+    <button name="twitter_sign_in" id="twitter_sign_in" value="twitter_sign_in" class="btn btn--block btn--social btn--twitter">
+        <img alt="" src="/i/twitter-icon-32.png" width="17" height="32">
+        [% loc('Log in with Twitter') %]
+    </button>
+    </div>
+  [% END %]
+    <div class="form-box hidden-nojs">
+        <button id="js-new-report-user-show" class="btn btn--social btn--social-email">[% loc('Log in with email') %]</button>
+    </div>
+  </div>
+    <button id="js-new-report-hide-sign-in" class="btn btn--provide-update hidden-js js-new-report-sign-in-shown">[% loc('Back') %]</button>
+    <div class="hidden-js js-new-report-user-shown">
+        <button id="js-new-report-user-hide" class="btn btn--provide-update js-new-report-user-shown js-new-report-sign-in-hidden">[% loc('Back') %]</button>
```

Comment:
Something to note for design is that this button appears when JS is turned off even though it's not used.

Reply:
Whoops, should be fixed.